crates/iota/tests/snapshots/shell_tests__new_tests__new_files.sh.snap

```diff
@@ -0,0 +1,31 @@
+---
+source: crates/iota/tests/shell_tests.rs
+description: tests/shell_tests/new_tests/new_files.sh
+---
+----- script -----
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This probably also needs to be updated and btw year is 2025 now :trollface:

Reply:
nuh uh :c